include/seahorn/seahorn.h

```diff
@@ -19,18 +19,31 @@ extern void __VERIFIER_error(void);
 */
 extern void __VERIFIER_assume(bool);
 
+extern void __VERIFIER_assert(bool);
+extern void __VERIFIER_assert_not(bool);
+extern void __VERIFIER_assert_if(bool, bool);
 /**
    Returns TRUE if \p offset number of bytes of \p ptr are allocated
 
     Requires support from the memory manager. Might be interpreted to always
     return TRUE or FALSE if the memory manager does not support it.
  */
 extern bool sea_is_dereferenceable(void *ptr, intptr_t offset);
+extern bool sea_assert_if(bool, bool);
```

Comment:
is `sea_assert_if` used? Remove if not used, make `void` if used

Reply:
used for "unit testing" sea.assert.if in tests/opsem. Fixed.